src/app/zap-templates/zcl/data-model/chip/smoke-co-alarm-cluster.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2008,2020 Silicon Labs.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP" />
+
+  <cluster>
+    <name>Smoke CO Alarm</name>
+    <domain>Closures</domain>
+    <code>0x005C</code>
+    <define>SMOKE_CO_ALARM_CLUSTER</define>
+    <description>This cluster provides an interface for observing and managing the state of smoke and CO alarms.</description>
+
+    <!-- Abbreviations used in descriptions -->
+    <tag name="SMOKE" description="Smoke Alarm" />
+    <tag name="CO" description="CO Alarm" />
```

Comment:
Normally these would go into a bitmap definition that defines the feature map.

Reply:
Added in the new commit.

Reply:
How do these get into the bitmap definition?